README.md

```diff
@@ -43,6 +51,7 @@ node findIssues.js
 
 This will update the [outputData](./OutputFiles/outputData.json) file with any issues that Mariner finds.
 
+The query will return only the data that you specify in the input files. GitHub will automatically paginate results in sets of 30. Different amounts can be specified up to 100.
```

Comment:
We should clarify here that Mariner will walk through the paginated results without needing to specify larger result sets.